services/core-web/src/components/mine/Incidents/MineIncidentTable.tsx

```diff
@@ -390,13 +401,10 @@ const MineIncidentTable = (props) => {
   );
 };
 
-MineIncidentTable.propTypes = propTypes;
-MineIncidentTable.defaultProps = defaultProps;
-
 const mapStateToProps = (state) => ({
   incidentDeterminationHash: getIncidentDeterminationHash(state),
   incidentStatusCodeHash: getIncidentStatusCodeHash(state),
-  incidentCategoryCodeHash: getIncidentCategoryCodeHash(state, false),
+  incidentCategoryCodeHash: getIncidentCategoryCodeHash(state),
```

Comment:
I remember adding this "false" here! (https://github.com/bcgov/mds/pull/2463)
I added it in order to be able to show "legacy" categories with "is_historic". Is that something that we're no longer doing?

Reply:
😕 
getIncidentCategoryCodeHash only accepts a single argument, so the false doesn't seem to be doing anything (Typescript was real mad about it).

Reply:
![image](https://github.com/bcgov/mds/assets/83598933/5320bd4e-be27-48c1-bebc-75995810cd95)

Reply:
It _looks_ like it's doing nothing, but ohhhhh that was an ordeal to figure out, it actually controls whether it filters the data, but I wouldn't be surprised if it's not correctly typed and throws an error. createSelector takes two arguments, state, and isActiveOnly, which defaults to true.
Have you gone to an incident with legacy categories to verify that they're not broken?

Reply:
It's slightly more complicated than I remembered. It goes through createSelectorWrapper and createSelector, but it's _getVisibiilityFilterOption_ that takes showActiveOnly. Which I'm not sure why it's typed as true | undefined, it should be boolean | undefined. Might be the problem there.